decisions/0016-use-elite2-api.md

```diff
@@ -0,0 +1,45 @@
+# 6. Use the Elite2 API to access NOMIS data
+
+Date: 2019-01-18
+
+## Status
+
+Accepted
+
+## Context
+
+Our main source of data on prisoners and prison staff is NOMIS. The
+allocation tool currently uses the Custody API, as decided in [ADR 0006](https://github.com/ministryofjustice/offender-management-architecture-decisions/blob/master/decisions/0006-use-the-custody-api-to-access-nomis-data.md), to retrieve
+data on both offenders and staff.
+
+Initial use of the Custody API has raised some issues. Problems exist
+with the locality of data, N+1 API requests, and data that is unavailable
+through the currently published endpoints. These issues are intrinsic to
+the current design of the Custody API, and it is unlikely that they will
+be resolved or mitigated in the short to medium term.
+
+The Elite2 API, a fork of the Custody API, is under more active development
```

Comment:
I didn't think Elite2 was a fork of the Custody API 🤔

Reply:
There's a youth justice API which is a fork of Elite2. Maybe Elite2 is also a fork of the original API? I thought (maybe assumed) that the Custody API was started from scratch. Maybe @Mjwillis can clarify?

Reply:
I've removed this from the ADR because it doesn't actually add anything and isn't really _that_ relevant.